pkgs/build-support/fetchgit/nix-prefetch-git

```diff
@@ -417,6 +417,7 @@ fi
 tmpHomePath="$(mktemp -d "${TMPDIR:-/tmp}/nix-prefetch-git-tmp-home-XXXXXXXXXX")"
 exit_handlers+=(remove_tmpHomePath)
 HOME="$tmpHomePath"
+ln -s /build/.netrc "$HOME/"
```

Comment:
Maybe use $NIX_BUILD_TOP or something here, instead of hardcoding /build?

Reply:
Good catch! This was more of a proposal to move this into the right direction.

Reply:
#259620